packages/grafana-ui/src/components/Table/FooterRow.tsx

```diff
@@ -71,10 +72,14 @@ function renderFooterCell(column: ColumnInstance, tableStyles: TableStyles, heig
   );
 }
 
-export function getFooterValue(index: number, footerValues?: FooterItem[]) {
+export function getFooterValue(index: number, footerValues?: FooterItem[], isCountAllSet?: boolean) {
   if (footerValues === undefined) {
     return EmptyCell;
   }
 
+  if (isCountAllSet) {
+    return FooterCell({ value: [{ Count: footerValues[index] as string }] });
```

Comment:
Can we properly narrow the type down, rather than just hoping for the best that the value is a string? 

We want to avoid adding _more_ exceptions to .betterer.results that we only have to go in and clean up later.

Reply:
Alternatively, rename this file to `.js` and remove all typescript and be prepared to justify the decision 😅